ui/raidboss/data/06-ew/trial/golbez-ex.ts

```diff
@@ -1,14 +1,225 @@
+import NetRegexes from '../../../../../resources/netregexes';
+import { UnreachableCode } from '../../../../../resources/not_reached';
+import Outputs from '../../../../../resources/outputs';
+import { callOverlayHandler } from '../../../../../resources/overlay_plugin_api';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
+import { PluginCombatantState } from '../../../../../types/event';
 import { TriggerSet } from '../../../../../types/trigger';
 
-export type Data = RaidbossData;
+export interface Data extends RaidbossData {
+  terrastormCount: number;
+  terrastormCombatantDirs: number[];
+}
+
+// Calculate combatant position in an all 8 cards/intercards
+const matchedPositionTo8Dir = (combatant: PluginCombatantState) => {
+  // Positions are moved up 100 and right 100
+  const y = combatant.PosY - 100;
+  const x = combatant.PosX - 100;
+
+  // Majority of mechanics center around three circles:
+  // NW at 0, NE at 2, South at 5
```

Comment:
Yeah, I agree that starting with `N=0` is better.

Reply:
I think in this case it's better to leave this helper in line with other helpers doing the same thing, and then clean it up during the consolidation effort so that it lines up with whatever option is used to adjust so that `N=0`.